packages/esm-active-visits-app/src/types/index.ts

```diff
@@ -26,3 +28,28 @@ export interface Identifier {
   preferred: boolean;
   voided: boolean;
 }
+
+export interface Visit {
+  uuid: string;
+  display?: string;
+  encounters: Array<OpenmrsResource>;
+  patient?: OpenmrsResource;
+  visitType: VisitType;
+  location?: Location;
+  startDatetime: string;
+  stopDatetime?: string;
+  attributes?: Array<OpenmrsResource>;
+  [anythingElse: string]: any;
```

Comment:
Was this a placeholder?

Reply:
Are you referring to: `[anythingElse: string]: any;`?
If so then this was just a contingency.

Reply:
Thank you for the clarity, will resolve this.